annotation/app/categories/resources.py

```diff
@@ -104,7 +106,7 @@ def get_child_categories(
 @router.post(
     "/search",
     status_code=status.HTTP_200_OK,
-    response_model=Page[Any],  # type: ignore
+    response_model=Page[Any],
```

Comment:
`Any` might be changed to pydantic model, which will be describe response schema. Right now it's not clear, what will be returned